cloudigrade/account/tasks.py

```diff
@@ -228,3 +235,55 @@ def run_inspection_cluster(messages, cloud='aws'):
         cluster=settings.HOUNDIGRADE_ECS_CLUSTER_NAME,
         taskDefinition=result['taskDefinition']['taskDefinitionArn'],
     )
+
+
+def persist_aws_inspection_cluster_results(inspection_result):
+    """
+    Persist the aws houndigrade inspection result.
+
+    Args:
+        inspection_result (dict): A dict containing houndigrade results
+    Returns:
+        None
+    """
+    RHEL_TAG = ImageTag.objects.filter(description='rhel').first()
```

Comment:
I think I understand why you capitalized this variable, but it still feels weird to me. The value being assigned to `RHEL_TAG` will _generally_ never change, which makes it _kind of_ like a constant, even though it has to be fetched each time and likely changes across environments. For consistency (and since `windows_tag` is lower-case elsewhere), I think this should be lower-cased so it looks like a regular variable.

That said, this does give me an idea to consider for the future but that we _don't_ need today. Since _within the same environment_ tags will probably never change, perhaps in the future we should create a helper function to get the tag objects (instead of going directly to the model class) which could memoize/cache the lookup to reduce redundant lookups.